inc/template-tags.php

```diff
@@ -216,13 +216,10 @@ function independent_publisher_mentions() {
 	 *
 	 * @since Independent Publisher 1.0
 	 */
-	function independent_publisher_posted_author() {
-		/**
-		 * This function gets called outside the loop (in header.php),
```

Comment:
@dshanske Please see the code comment here. We cannot use `get_the_author()` here because this function is called outside the loop and `get_the_author()` [can only be used inside the loop](https://codex.wordpress.org/Function_Reference/get_the_author).

Reply:
It is no longer called outside the loop. There is a second function that is called outside the loop that functions similarly.